content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -213,23 +223,20 @@ for production clusters!
 
 ### Explore other add-ons
 
-See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+See the [list of add-ons](/docs/concepts/cluster-administration/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization, and control of your Kubernetes cluster.
 
 ## Cleanup
 
 * To delete your cluster: `kops delete cluster useast1.dev.example.com --yes`
 
-## Feedback
-
-* Slack Channel: [#kops-users](https://kubernetes.slack.com/messages/kops-users/)
-* [GitHub Issues](https://github.com/kubernetes/kops/issues)
-
 {{% /capture %}}
 
 {{% capture whatsnext %}}
 
```

Comment:
Hyperlink to https://kops.sigs.k8s.io/ ?

Reply:
I have updated the link for `advanced usage` on line 239.